api/api.go

```diff
@@ -105,11 +105,15 @@ const (
 
 // Api clientserver Constants
 const (
-	OsdVolumePath   = "osd-volumes"
-	OsdSnapshotPath = "osd-snapshot"
-	OsdCredsPath    = "osd-creds"
-	OsdBackupPath   = "osd-backup"
-	TimeLayout      = "Jan 2 15:04:05 UTC 2006"
+	OsdVolumePath        = "osd-volumes"
+	OsdSnapshotPath      = "osd-snapshot"
+	OsdCredsPath         = "osd-creds"
+	OsdBackupPath        = "osd-backup"
+	OsdMigratePath       = "osd-migrate"
+	OsdMigrateStartPath  = OsdMigratePath + "/start"
```

Comment:
Why have these defined here? They should be defined in the routes instead as the others are.

Reply:
The way it is defined right now for the other paths is incorrect. The server and client either redefine them or hard code the paths. The server and client need to refer to them from a common place, which is this file

Reply:
Oh I see. Great point. In other words, there should be a common location for both to share. Please create an issue to fix it so that this style can be applied to all.